batcher/aligned-sdk/src/core/types.rs

```diff
@@ -265,3 +319,15 @@ pub enum Chain {
     Holesky,
     HoleskyStage,
 }
+
+fn get_payment_service_addr(chain_id: U256) -> Option<H160> {
+    match chain_id.as_u64() {
+        ANVIL_CHAIN_ID => H160::from_str("0x7969c5eD335650692Bc04293B07F5BF2e7A673C0").ok(),
+        HOLESKY_CHAIN_ID => H160::from_str("0x815aeCA64a974297942D2Bbf034ABEe22a38A003").ok(),
+        MAINNET_CHAIN_ID => {
+            //FIXME: Add the payment service address for mainnet
+            None
+        }
+        _ => None,
+    }
+}
```

Comment:
If we want to support different batchers I think the payment service address will need to be a field in the struct, and be passed by the client. I think most sdk methods use the payment service address now so it shouldn't be too hard to add

Reply:
I agree, I think we can add that in another PR